Added UYVY10 ( be ?) implementation

DTL2020 opened the pull request:
Some quick addition of EBU .yuv10 file format - full YUV 10bit in 16bit words interleaved samples with non-intel endian order. May be named as U210le or U210be.

Some quick and dirty implementation of only sort of U210 (little or big ? endian) format used by EBU for test patterns in .yuv10 files.

For complete list of new formats it required better programming re-write and may be make Endian-param separate from format naming.

DTL2020 committed:
Added U210 (le or be ?) implementation

Some quick addition of EBU .yuv10 file format - full YUV 10bit in 16bit words interleaved samples with non-intel endian order. May be named as U210le or U210be.

Asd-g commented on src/rawsource26.cpp:
```diff
@@ -292,7 +299,7 @@ RawSource::RawSource(const char *source, const int width, const int height,
         vi.num_frames = generate_index(index, rawindex, framesize, fileSize);
 
-        b = env->Allocate(vi.BytesFromPixels(vi.width * vi.height), 64, AVS_NORMAL_ALLOC);
+        b = env->Allocate(vi.BytesFromPixels(vi.width * vi.height * 2), 64, AVS_NORMAL_ALLOC); // *2 for up to 16 bit full interleaved formats
```
It's better to replace `* 2` with `* vi.ComponentSize()`
It should be added for non-v8 case too.

DTL2020 committed:
RE-done to simple UYVY10 additional format

Replaced write_packed_luma_chroma() with patch provided and left only one new pixel format UYVY10. For some unknown reason UYVY10be form not works - may be somewhere check for not allow low case lettters. The internal YUV422P10 also works well without converting 10 to 16bit.

DTL2020 committed:
Delete rawsource26.cpp

DTL2020 committed:
Delete write_frame.cpp

DTL2020 committed:
RE-done to simple one new format UYVY10

Applied provided patch to simple function write_packed_luma_chroma(). Also only one new format UYVY10 left. For some reason UYVY10be do not works - may be somewhere check for not allow lower case letters. Also Avisynth+ YUV422P10 looks like work well without converting 10 to 16 bit.

DTL2020 commented:
Applied provided patch to simple function write_packed_luma_chroma(). Also only one new format UYVY10 left. For some reason UYVY10be do not works - may be somewhere check for not allow lower case letters. Also Avisynth+ YUV422P10 looks like work well without converting 10 to 16 bit.
